Week1/homework/js2-exercises/5-catWalking.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Cat Walk</title>
+  </head>
+  <body>
+    <img style="position:absolute;" src="http://www.anniemation.com/clip_art/images/cat-walk.gif" />
+    <script >
+        const img = document.querySelector('img');
+        const srcCatWalking = 'http://www.anniemation.com/clip_art/images/cat-walk.gif';
+        const srcCatDancIng = 'https://media2.giphy.com/media/3mXcJgVhPxWZa/giphy.gif';
+        let left = 0
+
+        function catWalk() {
+
+            setInterval(frame, 50)
+            function frame() {
+
+                left = left > 1500 ? 0 : left;
```

Comment:
What if the screen size is less than `1500`? maybe look into getting the screen width using javascript for example using `window.screen.width` from webAPI
https://developer.mozilla.org/en-US/docs/Web/API/Screen/width